tests/unit/test_events.py

```diff
@@ -91,6 +94,16 @@ def pr_comment_empty_request(self):
         ) as outfile:
             return json.load(outfile)
 
+    @pytest.fixture()
+    def copr_build_results_start(self):
+        with open(DATA_DIR / "fedmsg" / "copr_build_start.json", "r") as outfile:
```

Comment:
"r" is not needed as it's [default mode](https://docs.python.org/3/library/functions.html#open)